.github/workflows/bench.yml

```diff
@@ -133,7 +133,7 @@ jobs:
             cp "$BENCH" ../binaries/neqo/
             # shellcheck disable=SC2086
             nice -n -20 setarch --addr-no-randomize cset proc --set=cpu23 --exec \
-              $BENCH -- --bench --baseline main | { grep -v '^cset' || test $? = 1; } | tee -a ../results.txt
+              $BENCH -- --bench --baseline-lenient main | { grep -v '^cset' || test $? = 1; } | tee -a ../results.txt
```

Comment:
This pull request introduces new benchmarks. They are not yet present on `main`.

Without `-lenient` `cargo bench` fails, as it can not find a `main` execution of those very benchmarks to compare against.

```
Baseline 'main' must exist before comparison is allowed; try --save-baseline main
```

```
  -b, --baseline <baseline>
          Compare to a named baseline. If any benchmarks do not have the specified baseline this command fails.
      --baseline-lenient <baseline-lenient>
          Compare to a named baseline. If any benchmarks do not have the specified baseline then just those benchmarks are not compared against the baseline while every other benchmark is compared against the baseline.
```

Reply:
Want to open an issue to remove this once we're stable?  I can see why having the ability to add benchmarks is useful.

It might pay to have a comment here if you keep it: `# --baseline-lenient allows us to add new benchmarks in pull requests`

Or plan to remove it: `# TODO: remove --baseline-lenient once #2891 is on main`

Reply:
:+1: would like to keep it. I expect to add more benchmarks in future pull requests. Added a comment in https://github.com/mozilla/neqo/pull/2891/commits/5c53f72ddbde8c2d5d81c7f3ab74b7927648ca47.